spec/tasks/whitehall_news_importer_spec.rb

```diff
@@ -65,41 +113,54 @@
     expect(Document.last.review_state).to eq("unreviewed")
   end
 
-  it "sets the correct publication state and review state when Whitehall document state is 'published'" do
+  it "sets the correct publication, review and has live states when Whitehall document state is 'published'" do
```

Comment:
Not a blocker, but could we just say 'sets the correct states'? The checks within the tests make it pretty clear which states we're checking :-).